### ISSUE TITLE
Can't install this plugin

### ISSUE THREAD

**Issue by jmchat** [codewithkyle/craft-lightkeeper#1]
We get this error during installation:

`ParseError: syntax error, unexpected ')'`

Any idea how to fix this?

**Comment by codewithkyle** [comment]
I've fixed an issue in v1.0.3 that was occurring during database migrations. Try installing the latest version and if the error persist please fill out the following information.

```
### Steps to reproduce

1.
2.

### Additional info

- Craft version:
- PHP version:
- Database driver & version:
- Plugins & versions:
```

**Comment by jmchat** [comment]
Yes it works!
